web/test/pages/profile.test.tsx

```diff
@@ -1900,130 +1900,112 @@ describe("profile", () => {
+          "testCert.pdf"
+        );
+      });
+
+      it("hides document links if they do not exist", () => {
+        const userData = generateUserData({
+          formationData: generateFormationData({
+            getFilingResponse: generateGetFilingResponse({ success: true }),
+          }),
+          profileData: generateProfileData({
+            businessPersona: "STARTING",
+            legalStructureId: "limited-liability-company",
+            documents: { certifiedDoc: "pp.zip", formationDoc: "whatever.pdf", standingDoc: "" },
+          }),
+        });
 
-    it("uses links from useDocuments hook", () => {
-      setMockDocumentsResponse({
-        formationDoc: "testForm.pdf",
-        certifiedDoc: "testCert.pdf",
-        standingDoc: "testStand.pdf",
-      });
-      const userData = generateUserData({
-        formationData: generateFormationData({
-          getFilingResponse: generateGetFilingResponse({ success: true }),
-        }),
-        profileData: generateProfileData({
-          businessPersona: "STARTING",
-          legalStructureId: "limited-liability-company",
-          documents: { certifiedDoc: "pz.zip", formationDoc: "whatever.pdf", standingDoc: "lol" },
-        }),
+        renderPage({ userData });
+        chooseTab("documents");
+
+        expect(screen.getByText(Config.profileDefaults.formationDocFileTitle)).toBeInTheDocument();
+        expect(screen.getByText(Config.profileDefaults.certificationDocFileTitle)).toBeInTheDocument();
+        expect(screen.queryByText(Config.profileDefaults.standingDocFileTitle)).not.toBeInTheDocument();
       });
-      renderPage({
-        userData,
-      });
-      chooseTab("documents");
-
-      expect(screen.getByText(Config.profileDefaults.formationDocFileTitle)).toHaveAttribute(
-        "href",
-        "testForm.pdf"
-      );
-      expect(screen.getByText(Config.profileDefaults.standingDocFileTitle)).toHaveAttribute(
-        "href",
-        "testStand.pdf"
-      );
-      expect(screen.getByText(Config.profileDefaults.certificationDocFileTitle)).toHaveAttribute(
-        "href",
-        "testCert.pdf"
-      );
     });
 
-    it("hides document links if they do not exist", () => {
-      const userData = generateUserData({
-        formationData: generateFormationData({
-          getFilingResponse: generateGetFilingResponse({ success: true }),
-        }),
-        profileData: generateProfileData({
-          businessPersona: "STARTING",
-          legalStructureId: "limited-liability-company",
-          documents: { certifiedDoc: "pp.zip", formationDoc: "whatever.pdf", standingDoc: "" },
-        }),
+    describe("if Oscar", () => {
+      it("has no document section", () => {
+        const userData = generateUserData({
+          profileData: generateProfileData({
+            businessPersona: "OWNING",
+          }),
+        });
+        renderPage({ userData });
+        expect(screen.queryByTestId("documents")).not.toBeInTheDocument();
```

Comment:
Changed test logic. I am pretty sure this is the correct logic, what we had before was old. Oscar can never have formation documents